tools/directus/schema-sync/data/schema/views_connexion.json

```
@@ -0,0 +1,235 @@
+{
+  "collection": "views_connexion",
+  "meta": {
+    "accountability": "all",
+    "archive_app_filter": true,
+    "archive_field": null,
+    "archive_value": null,
+    "collapse": "open",
+    "collection": "views_connexion",
+    "color": "#81E1EE",
+    "display_template": null,
+    "group": "views",
+    "hidden": false,
+    "icon": null,
+    "item_duplication_fields": null,
+    "note": null,
+    "preview_url": null,
+    "singleton": true,
+    "sort": 1,
+    "sort_field": null,
+    "translations": [
+      {
+        "language": "en-US",
+        "translation": "Connexion view"
+      },
+      {
+        "language": "fr-FR",
+        "translation": "Page de connexion"
+      }
+    ],
+    "unarchive_value": null,
+    "versioning": false
+  },
+  "schema": {
+    "name": "views_connexion"
+  },
+  "fields": [
+    {
+      "field": "id",
+      "type": "integer",
+      "meta": {
+        "collection": "views_connexion",
+        "conditions": null,
+        "display": null,
+        "display_options": null,
+        "field": "id",
+        "group": null,
+        "hidden": true,
+        "interface": "input",
+        "note": null,
+        "options": null,
+        "readonly": true,
+        "required": false,
+        "sort": 1,
+        "special": null,
+        "translations": null,
+        "validation": null,
+        "validation_message": null,
+        "width": "full"
+      },
+      "schema": {
+        "name": "id",
+        "table": "views_connexion",
+        "data_type": "integer",
+        "default_value": null,
+        "max_length": null,
+        "numeric_precision": 32,
+        "numeric_scale": 0,
+        "is_nullable": false,
+        "is_unique": true,
+        "is_indexed": false,
+        "is_primary_key": true,
+        "is_generated": false,
+        "generation_expression": null,
+        "has_auto_increment": true,
+        "foreign_key_table": null,
+        "foreign_key_column": null
+      }
+    },
+    {
+      "field": "LOGIN_BUTTON",
+      "type": "string",
+      "meta": {
+        "collection": "views_connexion",
+        "conditions": null,
+        "display": null,
+        "display_options": null,
+        "field": "LOGIN_BUTTON",
```

```suggestion
        "field": "BUTTON",
```

Le préfixe n'est pas utile, sachant qu'on est déjà dans la collection VIEWS, le préfixe c'est pour lier des éléments visuels, faire des blocs au sein d'une vue.

Est-ce que tu peux revoir ça pour toutes les vues stp ?